### ISSUE TITLE
strange warning on loadup

### ISSUE THREAD

**Issue by pavog** [pavog/craftconomy3#20]
From craftconomy3 created by [tara81](https://github.com/tara81) : greatman/craftconomy3#97  

[13:04:34] [Server thread/INFO]: [Craftconomy3] Loading the Database Manager
[13:04:34] [Server thread/WARN]: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[13:04:34] [Server thread/WARN]: SLF4J: Defaulting to no-operation (NOP) logger implementation
[13:04:34] [Server thread/WARN]: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

**Comment by pavog** [comment]
This issue seems to be caused by a not working db connection.
Maybe we should give a better error message on this.

**Comment by pavog** [comment]
Hopefully fixed in 3.5.0. Needs to be tested 🤞

**Comment by pavog** [comment]
Fixed this! Startup output is the following:

```
[Server thread/INFO]: [Craftconomy3] Enabling Craftconomy3 v3.5.2-SNAPSHOT
[Server thread/INFO]: [Craftconomy3] Starting up!
[Server thread/INFO]: [Craftconomy3] Loading the Configuration
[Server thread/INFO]: [Craftconomy3] Loading listeners.
[Server thread/INFO]: [Craftconomy3] Loading commands
[Server thread/INFO]: [Craftconomy3] Loading the Database Manager
[Server thread/WARN]: [Server thread] INFO com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
[Server thread/WARN]: [Server thread] INFO com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
[Server thread/INFO]: [Craftconomy3] Database Manager Loaded!
[Server thread/INFO]: [Craftconomy3] Loading the Currency manager.
[Server thread/INFO]: [Craftconomy3] Currency Manager Loaded!
[Server thread/INFO]: [Craftconomy3] Loading default settings.
[Server thread/INFO]: [Craftconomy3] Default settings loaded!
[Server thread/INFO]: [Craftconomy3] Loading the Account Manager
[Server thread/INFO]: [Craftconomy3] Account Manager Loaded!
[Server thread/INFO]: [Craftconomy3] World Group Manager loaded!
[Server thread/INFO]: [Craftconomy3] Ready!
```
